community/index.html

```diff
@@ -0,0 +1,128 @@
+            <a href="../community/index.html">Community</a>
+          </li>
+        </ul>
+      </div>
+      <div class="navDecoration navDecorationRight">
+        <div class="toolBar-button">
+          <div class="toolBar-icon icon-image-scene-data"></div>
+          <div class="icon-down-arrow"></div>
+          <div class="icon-text">Scene.002</div>
+          <div class="icon-down-arrow"></div>
+        </div>
+        <div class="toolBar-button toolBar-gizmo-toggle">
+          <div class="gizmo-toggle">
+            <input
+              type="checkbox"
+              id="gizmo-overlay"
+              name="gizmo-overlay"
+              value="gizmo"
+            />
+            <div class="gizmo-icon">
+              <label for="gizmo-overlay">
+                <div class="toolBar-icon icon-image-gizmo"></div>
+              </label>
+            </div>
+            <div class="gizmo-reveal-image"></div>
+          </div>
+          <div class="icon-down-arrow"></div>
+        </div>
+        <div class="toolBar-button toolBar-shading-toggle">
+          <input
+            type="radio"
+            id="shading-wireframe"
+            value="shading-wireframe"
+            name="shading-picker"
+            checked
+          />
+          <input
+            type="radio"
+            id="shading-solid"
+            value="shading-solid"
+            name="shading-picker"
+          />
+          <input
+            type="radio"
+            value="shading-material"
+            id="shading-material"
+            name="shading-picker"
+          />
+          <input
+            type="radio"
+            value="shading-rendered"
+            id="shading-rendered"
+            name="shading-picker"
+          />
+          <div class="shading-toggle-buttons toolBar-button">
+            <label class="toolBar-shading-option" for="shading-wireframe">
+              <div class="toolBar-icon icon-image-shading-wireframe"></div>
+            </label>
+            <label class="toolBar-shading-option" for="shading-solid">
+              <div class="toolBar-icon icon-image-shading-solid"></div>
+            </label>
+            <label class="toolBar-shading-option" for="shading-material">
+              <div class="toolBar-icon icon-image-shading-material"></div>
+            </label>
+            <label class="toolBar-shading-option" for="shading-rendered">
+              <div class="toolBar-icon icon-image-shading-rendered"></div>
+            </label>
+            <div class="icon-down-arrow"></div>
+          </div>
+          <div id="rendered-goat-container">
+            <div class="rendered-goat-image viewport-goat-wireframe"></div>
+            <div class="rendered-goat-image viewport-goat-solid"></div>
+            <div class="rendered-goat-image viewport-goat-material"></div>
+            <div class="rendered-goat-image viewport-goat-rendered"></div>
+          </div>
+        </div>
+      </div>
+    </nav>
```

Comment:
I really like what's being accomplished in the page with the code you've written here! I do have some recommendations on how this can be done a little simpler though.

- Try not to mix `camelCase` and `kabob-case` CSS class names. Pick one.
- I feel like a lot of things could have fewer total classes? Do something like putting a single class on the `<nav class="toolbar">`, and then write your CSS selectors like `.toolbar .icon`, to impact all icons inside the thing at the top level. Then like... most of these class names go away.

Fabulous work so far!!!